relay-config/src/config.rs

```diff
@@ -1336,6 +1336,43 @@ impl Default for Health {
     }
 }
 
+/// COGS configuration.
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(default)]
+pub struct Cogs {
+    /// Whether COGS measurements are enabled.
+    enabled: bool,
+    /// Granularity of the COGS measurements.
+    ///
+    /// Measurements are aggregated based on the granularity.
+    ///
+    /// Defaults to 60 (1 minute).
```

Comment:
From the default it's kinda obvious what the unit is, but docs should still mention it:
```suggestion
    /// Granularity of the COGS measurements.
    ///
    /// Measurements are aggregated based on the granularity, in seconds.
    ///
    /// Defaults to 60 (1 minute).
```

Reply:
IIUC the granularity also determines how often cogs are flushed. We should point that out.